pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -200,6 +200,8 @@ def get_serializer_context(self, *args, **kwargs):
         super_data = super().get_serializer_context()
         super_data["highest_versions"] = self.highest_versions_context
         super_data["lowest_versions"] = self.lowest_versions_context
+        repo_version = self.get_repository_version(self.kwargs["path"])
+        super_data["repo_version_last_updated"] = repo_version.pulp_last_updated
```

Comment:
This timestamp will move forward with each repo version published. That is what we want, but it also will move forward even when the CollectionVersions for this collection have not changed in between two repo versions. Is that ok? I'm a little unclear on how to query for the latter off the top of my head...

Reply:
yes, I couldn't think of a good way to check the CollectionVersion change between RepositoryVersions

Reply:
I think it would be too costly to make it precise, 
- latest_repo_version contains a new collection version? 
if no, get latest_repo_version - 1, ...